cloud/migrate-mst-cloud/entire-database.md

```diff
@@ -72,10 +73,11 @@ accordingly to avoid data loss.</highlight>
     SELECT timescaledb_post_restore();
     ```
 
-1. Decompress your data.
-
 </procedure>
 
+Your data is now in Timescale Cloud. If you compressed it before migration,
+[decompress chunks][decompress] as necessary for normal operation.
```

Comment:
I like to try and avoid result statements. Can we reframe this in the "for more information, see" format instead?